java/client/src/main/java/glide/api/commands/StringCommands.java

```diff
@@ -47,4 +48,90 @@ public interface StringCommands {
      *     Otherwise, return <code>OK</code>.
      */
     CompletableFuture<String> set(String key, String value, SetOptions options);
+
+    /**
+     * Increment the string representing a floating point number stored at <em>key</em> by
+     * <em>amount</em>. By using a negative increment value, the result is that the value stored at
```

Comment:
I'd prefer we use `<code></code>` tags instead of `<em>`.  
```suggestion
     * <code>amount</code>. By using a negative increment value, the result is that the value stored at
```